opencti-worker/src/worker.py

```diff
@@ -346,9 +376,31 @@ def data_handler(  # pylint: disable=too-many-statements, too-many-locals
                     raise ValueError("JSON data type is not a STIX2 bundle")
                 if len(content_json["objects"]) == 1 or no_split:
                     update = data["update"] if "update" in data else False
-                    imported_items = self.api.stix2.import_bundle_from_json(
-                        content, update, types, work_id
+                    imported_items, too_large_items_bundles = (
+                        self.api.stix2.import_bundle_from_json(
+                            content, update, types, work_id, self.objects_max_refs
+                        )
                     )
+                    if len(too_large_items_bundles) > 0:
+                        push_pika_connection = pika.BlockingConnection(
+                            self.pika_parameters
+                        )
+                        push_channel = push_pika_connection.channel()
+                        try:
+                            push_channel.confirm_delivery()
+                        except Exception as err:  # pylint: disable=broad-except
+                            self.worker_logger.warning(str(err))
+                        for too_large_item_bundle in too_large_items_bundles:
+                            self.worker_logger.warning(
+                                "Detected a bundle too large, sending it to dead letter queue...",
```

Comment:
could we log more information about the rejected bundle ?

Reply:
i don't know if there is some kind of id (from the bundle itself and/or rabbitmq message id) that would be useful